docusaurus.config.js

```diff
@@ -90,6 +90,11 @@ async function getConfig() {
               label: 'Composer Studio',
               position: 'left'
             },
+            {
+              to: '/docs/references',
+              label: 'References',
+              position: 'right'
+            },
```

Comment:
Based on what I've seen in other [docs sites](https://docusaurus.io/showcase), I think it should be "Reference" (singular)